modules/installation-configuration-parameters.adoc

```diff
@@ -515,6 +531,13 @@ ifdef::ibm-power[]
 |Determines the instruction set architecture of the machines in the pool. Currently, heteregeneous clusters are not supported, so all pools must specify the same architecture. Valid values are `ppc64le` (the default).
 |String
 endif::ibm-power[]
+
```

Comment:
Remove lines 534 - 540. 

Add ` ,ibm-power-vs` to lines 530 and 533

Reply:
Agree 👍